splice_machine/src/test/java/com/splicemachine/derby/impl/load/CaseSensitiveImportIT.java

```diff
@@ -43,11 +44,7 @@
     private TestConnection conn;
 
     @BeforeClass
-    public static void beforeClass() throws Exception {
-        cleanSchema(SCHEMA_NAME, methodWatcher);
```

Comment:
Has schema cleaning before execution been moved somewhere else?
Should it be done for any test that creates a new schema?
cleanSchema is a private method not used anywhere else. Should be removed.

Reply:
Yes, every test case will clean the created schema within the ClassRule of SpliceSchemaWatcher. I'll remove the unused `cleanSchema`.